test/help.test.js

```diff
@@ -45,6 +46,22 @@ describe('loopback generator help', function() {
     process.env.SLC_COMMAND = undefined;
   });
 
+  it('print right help message for each generator', function() {
+    process.env.SLC_COMMAND = 'loopback --help';
+    var names = ['acl', 'app', 'boot-script', 'datasource', 'export-api-def',
+    'middleware', 'model', 'property', 'relation', 'remote-method', 'swagger'];
+
+    names.forEach(function(name) {
+      var gen = givenGenerator(name, ['--help']);
+      var helpText = gen.help();
+      var helpFileName = 'loopback_' + name + '_help.txt';
+      var helpFilePath = '../fixtures/' + helpFileName;
+      var output = fs.readFileSync(helpFilePath, 'utf8');
+      assert.equal(output, helpText);
+    });
+    process.env.SLC_COMMAND = undefined;
```

Comment:
This won't be run when the assertion above fails. Please move this line into a beforeEach or afterEach hook.